### ISSUE TITLE
Pad/collaborative document

### ISSUE THREAD

**Issue by evamarik** [evamarik/2018-11-12-cologne#4]
Sollen wir wie in Marburg https://hackmd.io/ verwenden oder etwas anderes?

**Comment by katrinleinweber** [comment]
Wegen der direkten Konfrontation mit Markdown: +1 für HackMD.io von mir

**Comment by konrad** [comment]
Ja, bin auch dafür und hatte in vorherigen Workshops immer gute Erfahrung damit gemacht.

**Comment by evamarik** [comment]
Ok, habe ich ein der Workshop-Seite eingefügt.
